sklearn/tests/test_common.py

```diff
@@ -45,14 +45,17 @@
 
 import sklearn
 
+# make it possible to discover experimental estimators when calling `all_estimators`
+from sklearn.experimental import enable_iterative_imputer  # noqa
+from sklearn.experimental import enable_halving_search_cv  # noqa
```

Comment:
```suggestion
from sklearn.experimental import enable_halving_search_cv  # noqa
from sklearn.experimental import enable_hist_gradient_boosting  # noqa
```
While we are at it, also add hist gradient boosting?

Reply:
`HistGradientBoosting*` does not need the experimental import anymore it is there just for backward-compatibility (and warns that it is not needed anymore) ...

Reply:
Ah ok. I should have run the code :D